closure-deps/package.json

```diff
@@ -33,11 +33,12 @@
   },
   "homepage": "https://developers.google.com/closure/library/",
   "devDependencies": {
-    "google-closure-library": "^20180805.0.0",
-    "jasmine": "^2.9.0"
+    "google-closure-library": "^20180716.0.0",
```

Comment:
This is probably fine for now since it isn't too old. I'll make a PR in the future to have the closuremakedeps_test ignore *_test.js files in the expected output, and then we can use the head library again.

Reply:
fixed https://github.com/google/closure-library/commit/8e98199d401884257bc0758f3a35e173d0c178b5